data/json/npcs/items_generic.json

```diff
@@ -582,7 +582,7 @@
       [ "grip", 1 ],
       [ "gum", 8 ],
       [ "gun_crossbow", 1 ],
-      [ "gunpowder", 5 ],
+      [ "gunpowder", 5000 ],
```

Comment:
Is this necessary? I think it'd increase spawn rates for gunpowder by a lot, which likely has nothing to do with gunpowder having smaller charges now.

Reply:
It isn't at all, nice catch. I was searching in files for gunpowder and didn't check to see if that was weight or amount of items.